### PR TITLE
Add time zone text to projects/project-name page

### LINKED ISSUE #1171: The cards under proiects/website url does not mention the timezone
### Overview
Currently the hackforla project cards displaying the events meeting times does not state that the meetings times are being displayed in the users local time. This can cause confusion to people who are being onboarded, because the general assumption is since its hack4la all meeting times that are displayed are shown in PST.

### Action Items
- [ ] Discuss adding some kind of paragraph like this image that shows the user that the times displayed are in their local time zone
image

### Resources/Instructions
https://www.hackforla.org/projects/website
<details>
<summary>No Time Zone Specified in The Card</summary>

image

</details>

### PR DESCRIPTION
**Pull request by akibrhast**
Closes #1171 

- [x] Added time zone text to projects/project-name page
- [x] Styling consistency matches of that in events page time zone text

<details>
<summary>Timezone Text Image</summary>

image

</details>

### CODE REVIEW

**Comment by daniellex0** [comment]
Looks good, almost there but the styling is supposed to be slightly different - normal font-weight and font-size 15px if possible. Can that be adjusted? Maybe this one could be given its own class- like- 

```

.userTimeZone--project-page {
  font-size: 15px;
}
```

But with the same ID so that it's still attached to the userTimeZone logic..?

Sorry I know this seems minor, but it's for the visual hierarchy of the page so that viewers focus more on the meeting times themselves first.

Here's the Figma screenshot for reference:

Screen Shot 2021-03-16 at 5 11 08 PM

**Comment by akibrhast** [comment]
@DanielekPark 
image

File: _layouts/project.html [hunk 1]
```diff
@@ -438,8 +439,16 @@ <h4 class='project-page-card-title'>Resources</h4>
     console.log(err);
 })
 
+/**
+ * Displays/Inserts the user's time zone in the DOM
+*/
+function insertTimeZoneText(){
+    let timeZoneText  = `Times are in your local time zone: ${Intl.DateTimeFormat().resolvedOptions().timeZone}`;
```
Comment (drubgrubby) [inline _layouts/project.html]: What happens if the code fails and doesn't retrieve a time zone?

**Review by drubgrubby — COMMENTED** [review]
@akibrhast - Can this code, and the code for the the other pages that display times, be written once and pulled into each of the pages?

**Review by daniellex0 — APPROVED** [review]
Looks good! Approved on my end, pending David's comment

**Review by drubgrubby — APPROVED** [review]
Looks good.  I somehow feel better that all of these pages are showing the time zone, and yet the code is being reused. 👍

**Pull request merged**